keyboards/metamechs/timberwolf/timberwolf.h

```diff
@@ -0,0 +1,175 @@
+    K000, K010, K001, K011, K002, K012, K003, K013, K004, K014, K005, K015, K006, K016,       K017, K008, K018, \
+    K020, K030, K021, K031, K022, K032, K023, K033, K024, K034, K025, K035, K026, K036, K027, K037, K028, K038, \
+    K040, K050, K041, K051, K042, K052, K043, K053, K044, K054, K045, K055, K046, K056, K047,       K048, K058, \
+    K060, K070, K061, K071, K062, K072, K063, K073, K064, K074, K065, K075, K066,       K067,       K068, K078, \
+    K080, K090,       K091, K082, K092, K083, K093, K084, K094, K085, K095, K086, K096, K087, K088,       K098, \
+    K100, K110, K101, K111,                   K113,             K105, K115, K106,       K107, K117, K108        \
+) { \
+    { K000, K001, K002, K003, K004, K005, K006, ____, K008 }, \
+    { K010, K011, K012, K013, K014, K015, K016, K017, K018 }, \
+    { K020, K021, K022, K023, K024, K025, K026, K027, K028 }, \
+    { K030, K031, K032, K033, K034, K035, K036, K037, K038 }, \
+    { K040, K041, K042, K043, K044, K045, K046, K047, K048 }, \
+    { K050, K051, K052, K053, K054, K055, K056, ____, K058 }, \
+    { K060, K061, K062, K063, K064, K065, K066, K067, K068 }, \
+    { K070, K071, K072, K073, K074, K075, ____, ____, K078 }, \
+    { K080, ____, K082, K083, K084, K085, K086, K087, K088 }, \
+    { K090, K091, K092, K093, K094, K095, K096, ____, K098 }, \
+    { K100, K101, ____, ____, ____, K105, K106, K107, K108 }, \
+    { K110, K111, ____, K113, ____, K115, ____, K117, ____ } \
+}
+
+#define LAYOUT_prime_iso( \
+    K000, K010, K001, K011, K002, K012, K003, K013, K004, K014, K005, K015, K006, K016,       K017, K008, K018, \
+    K020, K030, K021, K031, K022, K032, K023, K033, K024, K034, K025, K035, K026, K036, K027, K037, K028, K038, \
+    K040, K050, K041, K051, K042, K052, K043, K053, K044, K054, K045, K055, K046, K056,             K048, K058, \
+    K060, K070, K061, K071, K062, K072, K063, K073, K064, K074, K065, K075, K066, K076, K067,       K068, K078, \
+    K080, K090, K081, K091, K082, K092, K083, K093, K084, K094, K085, K095, K086, K096, K087, K097,       K098, \
+    K100, K110, K101, K111,                   K113,             K105, K115, K106,       K107, K117, K108        \
+) { \
+    { K000, K001, K002, K003, K004, K005, K006, ____, K008 }, \
+    { K010, K011, K012, K013, K014, K015, K016, K017, K018 }, \
+    { K020, K021, K022, K023, K024, K025, K026, K027, K028 }, \
+    { K030, K031, K032, K033, K034, K035, K036, K037, K038 }, \
+    { K040, K041, K042, K043, K044, K045, K046, ____, K048 }, \
+    { K050, K051, K052, K053, K054, K055, K056, ____, K058 }, \
+    { K060, K061, K062, K063, K064, K065, K066, K067, K068 }, \
+    { K070, K071, K072, K073, K074, K075, K076, ____, K078 }, \
+    { K080, K081, K082, K083, K084, K085, K086, K087, ____ }, \
+    { K090, K091, K092, K093, K094, K095, K096, K097, K098 }, \
+    { K100, K101, ____, ____, ____, K105, K106, K107, K108 }, \
+    { K110, K111, ____, K113, ____, K115, ____, K117, ____ } \
+}
```

Comment:
These should be `LAYOUT_iso/ansi` in `keyboards/metamechs/timberwolf/prime/prime.h`.

Reply:
this and the other changes are how i originally had it, but zvecr told me to change it since theyre all on the same pcb

Reply:
Ah okay, just read through all the comments and it seems all versions use the same PCB.
Sorry about my inattention.

I'd reeeaaally like to see the PCB, though.

Reply:
https://cdn.discordapp.com/attachments/728572418481979403/729631871297912852/unknown.png here's a taste :D

Reply:
The Swiss would like their cheese back. :D